src/arns.lua

```diff
@@ -474,16 +474,21 @@ function arns.calculateYearsBetweenTimestamps(startTimestamp, endTimestamp)
 	return yearsRemainingFloat
 end
 
+--- Asserts that a name is a valid ARNS name
+--- @param name string The name to check
+function arns.assertValidArNSName(name)
+	assert(type(name) == "string", "Name is required and must be a string.")
+	assert(#name >= constants.MIN_NAME_LENGTH and #name <= constants.MAX_NAME_LENGTH, "Name pattern is invalid.")
```

Comment:
Nit: could have a more descriptive error. Same for the regex pattern below. Could even have the regex string in the error message.

Reply:
[f4db59a](https://github.com/ar-io/ar-io-network-process/pull/216/commits/f4db59a3098c6ab86484ec7beeacb9d08412cd92)